properties-files/openshiftv4_on_osp.properties

```diff
@@ -0,0 +1,54 @@
+ORCHESTRATION_USER=stack
+ORCHESTRATION_HOST=
+SSHKEY_TOKEN=
+OPENSHIFT_INSTALL=true
+SCALE_CI_BUILD_TRIGGER=true
+SCALE_CI_BUILD_TRIGGER_URL=
+KUBECONFIG_PATH=/home/stack/ostest/auth/kubeconfig
+OPENSHIFT_INSTALL_APIVERSION=v1
+OPENSHIFT_INSTALL_SSH_PUB_KEY_FILE=/root/.ssh/id_rsa.pub
+OPENSHIFT_INSTALL_PULL_SECRET=
+OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN=
+OPENSHIFT_INSTALL_IMAGE_REGISTRY=
+OPENSHIFT_INSTALL_REGISTRY_TOKEN=
+OPENSHIFT_TOGGLE_INFRA_NODE=
+SCALE_LAB_CLOUD_NAME=
+ANSIBLE_SSH_PASSWORD=
+OPENSTACK_VERSION=
+OSP_EXTERNAL_GATEWAY=
+OSP_EXTERNAL_NET_CIDR=
+OSP_EXTERNAL_ALLOCATION_POOLS_START=
+OSP_EXTERNAL_ALLOCATION_POOLS_END=
+OSP_EXTERNAL_INTERFACE_DEFAULT_ROUTE=
+OPENSHIFT_BASE_DOMAIN=
+OPENSHIFT_CLUSTER_NAME=
+OPENSHIFT_RELEASE=4.4.6
+OPENSHIFT_MASTER_COUNT=3
+OPENSHIFT_WORKER_COUNT=5
+OPENSHIFT_MASTER_INSTANCE_TYPE=nvme
+OPENSHIFT_WORKER_INSTANCE_TYPE=m4.xlarge
+OPENSHIFT_MASTER_ROOT_VOLUME_SIZE=64
+OPENSHIFT_MASTER_ROOT_VOLUME_TYPE=gp2
+OPENSHIFT_MASTER_ROOT_VOLUME_IOPS=0
+OPENSHIFT_WORKER_ROOT_VOLUME_SIZE=64
+OPENSHIFT_WORKER_ROOT_VOLUME_TYPE=gp2
+OPENSHIFT_WORKER_ROOT_VOLUME_IOPS=0
+OPENSHIFT_CIDR=10.128.0.0/14
+OPENSHIFT_MACHINE_CIDR=10.0.0.0/16
+OPENSHIFT_NETWORK_TYPE=
+OPENSHIFT_SERVICE_NETWORK=172.30.0.0/16
+OPENSHIFT_HOST_PREFIX=23
+OPENSHIFT_POST_INSTALL_POLL_ATTEMPTS=600
+OPENSHIFT_TOGGLE_WORKLOAD_NODE=true
+MACHINESET_METADATA_LABEL_PREFIX=machine.openshift.io
+OPENSHIFT_INFRA_NODE_INSTANCE_TYPE=m4.xlarge
+OPENSHIFT_WORKLOAD_NODE_INSTANCE_TYPE=m4.xlarge
+OPENSHIFT_WORKLOAD_NODE_VOLUME_SIZE=64
+OPENSHIFT_WORKLOAD_NODE_VOLUME_TYPE=gp2
+OPENSHIFT_WORKLOAD_NODE_VOLUME_IOPS=0
+OPENSHIFT_PROMETHEUS_RETENTION_PERIOD=15d
+OPENSHIFT_PROMETHEUS_STORAGE_CLASS=gp2
```

Comment:
Are these defaults valid on OSP environment? Also what flavors do we make available in OSP?

Reply:
no ravi, will change the available values